torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -713,23 +748,11 @@ def _lower_quantized_binary_op(
     modules = dict(model.named_modules(remove_duplicate=False))
     for n in model.graph.nodes:
         # Step 0: Find nodes that match this pattern (dequantize - ref module - quantize)
-        # We search for the pattern backwards, starting with the quantize node
-        # Quantize node args: (func, scale, zp, dtype)
-        if n.op != "call_function" or n.target != torch.quantize_per_tensor:
-            continue
-        q_node = n
-        (bop_node, scale_node, zero_point_node, _) = q_node.args
-        # Handle cases where the module is wrapped in a ReLU
-        if bop_node.op == "call_function" and bop_node.target in (torch.nn.functional.relu, torch.relu) or\
-                bop_node.op == "call_module" and type(modules[str(bop_node.target)]) == torch.nn.ReLU:
-            relu_node = bop_node
-            bop_node = relu_node.args[0]
-        else:
-            relu_node = None
-        if should_skip_lowering(bop_node, qconfig_map):
-            continue
-        if bop_node.op != "call_function" or bop_node.target not in binary_ops_to_lower:
+        (q_node, relu_node, bop_node) = _match_static_pattern(
+            n, modules, qconfig_map, binary_ops_to_lower, skip_dequantize_matching=True)
```

Comment:
why do we skip matching dequantize?

Reply:
Because we also allow these patterns:
```
add(dequantize(x), y)
add(x, dequantize(y))
```
Since it's logic specific to binary ops I feel it's better to check dequantize node below (L761) instead of in a common function

Reply:
I see, can we try to match dequantize if it is a Node? in the add scalar cases the non dequantized value would be a constant I think

Reply:
Ok, let me try